src/propagate_constant.cpp

```diff
@@ -83,7 +84,12 @@ void propagate_constant::apply(module& m) const
     // Compute literals in parallel
     std::vector<instruction_ref> const_instrs_vec{const_instrs.begin(), const_instrs.end()};
     std::vector<argument> literals(const_instrs_vec.size());
-    simple_par_for(const_instrs_vec.size(), 1, [&](const auto i) {
+    std::size_t n = 1;
```

Comment:
This variable should be named `grainsize` to make it clear its the grainsize.

Reply:
Since this PR was merged, I have made this change in a follow up PR: https://github.com/ROCm/AMDMIGraphX/pull/2993